frontend/packages/pipelines-plugin/src/components/pipelineruns/list-page/PipelineRunRow.tsx

```diff
@@ -85,15 +87,20 @@ const PipelineRunRow: React.FC<RowFunctionArgs<PipelineRunKind>> = ({ obj, custo
         <PipelineRunVulnerabilities pipelineRun={obj} condensed />
       </TableData>
       <TableData className={tableColumnClasses.status}>
-        <PLRStatus obj={obj} taskRuns={PLRTaskRuns} />
+        <PLRStatus obj={obj} taskRuns={PLRTaskRuns} iconOnly />
       </TableData>
       <TableData className={tableColumnClasses.taskStatus}>
         <LinkedPipelineRunTaskStatus pipelineRun={obj} taskRuns={PLRTaskRuns} />
       </TableData>
       <TableData className={tableColumnClasses.started}>
-        <Timestamp timestamp={obj.status && obj.status.startTime} />
+        <div className="opp-pipeline-run-list__started-timestamp">
+          <Timestamp timestamp={obj.status && obj.status.startTime} omitSuffix />{' '}
+          <div>&nbsp; {t('pipelines-plugin~ago')}</div>
```

Comment:
This &nbsp; and the space shows two spaces here:

![image](https://github.com/openshift/console/assets/139310/d8895415-c19f-46d1-949a-7fee159c33e9)